src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java

```diff
@@ -105,17 +216,21 @@ public static JsonObject flagToFeature(final CheckFlag flag,
      * @return {@link JsonObject} created from {@link CheckFlag}
      */
     public static JsonObject flagToJson(final CheckFlag flag,
-            final Map<String, String> additionalProperties)
+            final Map<String, Object> additionalProperties)
```

Comment:
Is there a reason you switched to `Object` here?

Reply:
It was modified when I refactored the code to use  `GeometryWithProperties` that uses Map<String, Object>' instead of Map<String, String> used in the deprecated class `LocationIterableProperties`. But looks like this method needn't be changed. Good catch!